src/applications/vaos/components/Breadcrumbs.unit.spec.jsx

```diff
@@ -11,7 +11,9 @@ describe('VAOS Component: Breadcrumbs', () => {
       initialState,
       path: url,
     });
-    const navigation = screen.getByRole('navigation', { name: 'Breadcrumbs' });
+    const navigation = screen.getByTestId('vaos-breadcrumbs', {
+      name: 'Breadcrumbs',
+    });
```

Comment:
`getByTestId` does not need a second parameter, `{name: 'Breadcrumbs'}`  

We can delete the second param since it's leftover code from `getByRole`